tests/Unit/ConfigTest.php

```diff
@@ -55,3 +55,21 @@
         ])
         ->and($config->whitelistedPaths)->toBe([]);
 });
+
+it('can set ignore words', function (): void {
+    Config::flush();
+    $filePath = __DIR__.'/../Fixtures/peck.json';
+    Config::resolveConfigFilePathUsing(
+        fn (): string => $filePath,
+    );
+
+    Config::init();
+    $config = Config::instance();
+
+    expect($config->whitelistedWords)->toBe([]);
+
+    $config->ignoreWords(['laravel', 'paravel']);
+    Config::flush();
+
+    expect($config->whitelistedWords)->toBe(['laravel', 'paravel']);
```

Comment:
Don't you need to remove the file again after the assertion?

Reply:
Thanks @c0nst4ntin that actually highlighted a problem with the test so there was a false positive. This has been refactored.